.gitignore

```diff
@@ -1,6 +1,8 @@
 # Eclipse artifacts
 .project
 .pydevproject
+#Vagrant
```

Comment:
It would be really good if the source was kept clean - can we (cd $GOPATH/out ) - where 
we generate artifacts - before starting vagrant ?